app/frontend/src/lib/autocomplete.view.js

```diff
@@ -89,7 +89,7 @@ export const getOptionIndex = el => parseInt(el.getAttribute('aria-posinset'), 1
 export const getFocusedOption = container => container.getElementsByClassName('autocomplete__option--focused')[0];
 
 export const getOptionHtml = (refinement) => {
-    return (hit, index, options) => `<li class="autocomplete__option" id="autocomplete__input__option--${index}" role="option" tabindex="${index}" aria-setsize="${options.length + 1}" aria-posinset=${index} data-location="${hit.toLowerCase()}">${highlightRefinement(hit, refinement)}</li>`;
+    return (hit, index, options) => `<li class="autocomplete__option" id="autocomplete__input__option--${index}" role="option" tabindex="0" aria-setsize="${options.length + 1}" aria-posinset=${index} data-location="${hit.toLowerCase()}">${highlightRefinement(hit, refinement)}</li>`;
```

Comment:
Can you add `tabIndex: 0` to the `#vacancies-hits` element in `app/views/vacancies/index.html.haml`too?

Reply:
i dont think that makes sense, the vacancies hits part of the page follows the natural tab index? am i misunderstanding your comment?

Reply:
plus also i think if we are too look at the a11y of the results part of the page, that is another task, but the it is not bad in that part of the page imo